src/pages/Main/Main.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import NavBar from 'molecules/NavBar/NavBar';
 import SemesterTabs from 'molecules/SemesterTabs/SemesterTabs';
 import Calendar from 'atoms/Calendar/Calendar';
+import Dropdown from '../../atoms/Calendar/Dropdown';
```

Comment:
Can you use absolute imports ie `import Dropdown from 'atoms/Calendar/Dropdown';`